### DISCUSSION TITLE
Failed to build gem native extension You have to install development tools first

### DISCUSSION THREAD

**Discussion by tomy-shen** [docker-library/redmine#335] [General]
when i install redmine_crm plugin，system shows 'Failed to build gem native extension.... You have to install development tools first'

The current docker image is missing ubuntu-dev-tools（apt-get install ubuntu-dev-tools），Is it possible to consider packaging it into the docker image?

**Comment by LaurentGoderre** [comment]
To install native extensions, you need to extend this image with your own. Shipping the dev tools by default would require all musers to download a much bigger image for tools they might not need.
